packages/back-end/src/services/experiments.ts

```diff
@@ -1167,14 +1187,32 @@ export async function createSnapshot({
     integration,
     useCache,
   );
-  await queryRunner.startAnalysis({
-    snapshotType: type,
-    snapshotSettings: data.settings,
-    variationNames: experiment.variations.map((v) => v.name),
-    metricMap,
-    queryParentId: snapshot.id,
-    factTableMap,
-  });
+  if (startAnalysis ?? true) {
```

Comment:
Line 1134 above means it still gets put into the DB in the "running" state. Doesn't that still cause the main results to think this snapshot is running?